.circleci/config.yml

```diff
@@ -81,7 +81,7 @@ commands:
           # In the cache key:
           #   - .Environment.CIRCLE_JOB: We do separate tox environments by job name, so caching and restoring is
           #                              much faster.
-          key: tox-cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "tox.ini" }}
+          key: tox-cache-{{ .Environment.CIRCLE_JOB }}-{{ checksum "tox.ini" }}-{{ checksum "riotfile.py" }}-{{ checksum "setup.py" }}
```

Comment:
do we need riotfile here?

Reply:
we don't cache riot venvs

Reply:
so you should remove it right? 😁

Reply:
sorry, let me clarify, why do we have `checksum "riotfile.py"` as part of this cache key?

Reply:
sorry, pushed this by accident.